code.user.js

```diff
@@ -520,13 +520,30 @@
 
     SteamMarket.prototype.getGooValue = function(item, callback) {
         try {
+            var appid = item.market_fee_app;
+
+            for (const action of item.owner_actions) {
+                if (!action.link || !action.link.startsWith('javascript:GetGooValue')) {
+                    continue;
+                }
+
+                var rgMatches = action.link.match( /GetGooValue\( *'?(?:[%a-z0-9]+)'? *, *'?(?:[%a-z0-9]+)'? *, *'?(?<appid>[0-9]+)'?/ );
```

Comment:
Can we make this a bit more readable? I think this should be OK:
```
GetGooValue\(\s*'.*?'\s*,\s*'.*?'\s*,\s*(?<appid>[0-9]+)\s*,
```

Reply:
I've effectively copied it from Valve's code, they're also regexing it. The only thing I changed was add `%a-z` (%assetid% is not replaced) and cut off parsing the remaining parameters.